.travis.yml

```diff
@@ -1,61 +1,65 @@
+---
```

Comment:
difficult to tell if this new config is working because tests currently fail

Reply:
It's probably working. I've done the work to add Github Actions in this PR as well. I recognize that it's running wildly large, but... what the heck! Why not?

Reply:
with github actions in place I'd recommend we just delete the travis config.

Reply:
I left Travis in place as Github Actions weren't building on this PR, thus I needed some level of confidence that this PR works in more pristine environments. How about I drop the Travis config just before we "make rocket go"?